### PR TITLE
Docs: Add Release Documentation

### PR DESCRIPTION
**Pull request by omgaz**
Adds release process.

### CODE REVIEW

**Comment by lteacher** [comment]
lgtm

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Release
+
+To release we tag and publish to NPM using [SemVer](http://semver.org/).
+
+We check all commits since previous release and tag appropriatley:
```
Comment (bchew) [inline RELEASE.md]: `appropriately`

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- No version: Docs, Build
+- Patch version: Fix
+- Minor: New, Update, Upgrade
+- Major: Breaking
+
+If at lease one breaking commit since last release we tag as *major*.
```
Comment (bchew) [inline RELEASE.md]: `least`

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Else, if at lease one new, update or upgrade commit we tag as *minor*.
```
Comment (bchew) [inline RELEASE.md]: `least`

**Comment by bchew** [comment]
Merging :rocket:

**Pull request merged**